Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -133,7 +181,10 @@ def StateActivity(self, state, mouse1, mouse2):
             self.mouse_reward = "-"
             self.opponent_reward = "-"
 
-            self.trial_logger.log_trial_data(self.numcompletedtrial, "Return Abort", self.opponent_choice, self.mouse_choice,self.mouse_reward, self.opponent_reward)
+            self.trial_logger.log_trial_data(self.numcompletedtrial, "Completed Trial", self.opponent_choice,
```

Comment:
ive already changed that in the latest version

Reply:
This does not really help me. You need to make the change on this branch which we are reviewing. Otherwise I am just wasting my time with all these reviews.